src/java/htsjdk/samtools/SamFileValidator.java

```diff
@@ -90,7 +90,7 @@
     private Set<Type> errorsToIgnore = EnumSet.noneOf(Type.class);
     private boolean ignoreWarnings = false;
     private boolean bisulfiteSequenced = false;
-    private boolean validateIndex = false;
+    private ValidationStringency indexValidationStringency = ValidationStringency.SILENT;
```

Comment:
I'm going to change the use of ValidationStringency here, but please note that changing it to "SILENT" was just implementing the "validateIndex = false;" in the line above.  I think the default behavior would have been the same.